### PR TITLE
Move ref data controller to Api Controller

### PR DESCRIPTION
**Pull request by shakisparki**
Move refdata controllers to Api controllers

### CODE REVIEW

File: src/Directory/Biobanks.Web/ApiControllers/AccessConditionsController.cs [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+            //Everything went A-OK!
+            return Json(new
+            {
+                msg = $"The access condition \"{model.Description}\" was deleted successfully.",
+                type = FeedbackMessageType.Success
+            });
+        }
+
+        private IHttpActionResult JsonModelInvalidResponse(ModelStateDictionary state)
```
Comment (shakisparki) [inline src/Directory/Biobanks.Web/ApiControllers/AccessConditionsController.cs:150]: better to move this to separate controller class?

Comment (beforan) [reply src/Directory/Biobanks.Web/ApiControllers/AccessConditionsController.cs:150]: It's more like a helper method but that they all use. Either chuck it in some kind of utils class, or create a base api controller with it on, I guess.

**Comment by beforan** [comment]
This PR needs to wait for the directory codebase migration to GitHub to complete. then we can rebase and go from there.

**Comment by Flemingjp** [comment]
Closed in favour of its rebase

**Pull request closed without merging**